addons/sale/models/sale_order.py

```diff
@@ -985,8 +991,9 @@ def _prepare_invoice(self):
             'invoice_line_ids': [],
         }
 
-    def action_view_invoice(self):
-        invoices = self.mapped('invoice_ids')
+    def action_view_invoice(self, invoices=False):
+        if not invoices:
+            invoices = self.mapped('invoice_ids')
```

Comment:
why is this change necessary ?
AFAIK, the invoices created by the wizard should be linked to the SOs anyway.

Reply:
This is to (only) show the newly created down payment after the wizard. Imo i make sens to display the down payment instead of a list of the dowmnpayment + existing invoices

Reply:
Well, imo, the down payment invoice(s) are the first ones created, and it's not a problem to clearly see all the existing invoices :thinking:.
Did the PO ask for this change ?

Reply:
No it's an initiative. The idea is, in the case their already exist one down payment, and you go through the wizard to create a second down payment, if found i more user friendly to display the newly created down payment in form view instead of displaying a list, where the user then must click on the new down payment to view it.